README.md

```diff
@@ -146,80 +146,9 @@ For more details, see the [update system documentation](sources/updater/).
 
 There are several ways of updating your Bottlerocket hosts:
 
-#### Update API
-
-The [Bottlerocket API](#api) allows you to update and reboot your host with simple API calls.  You can change [settings](#updates-settings) to control which updates are selected.
-
-In general, the process of using the update API looks like this.  You refresh the list of known updates, then apply one to the system.  Calls to `/updates/status` will tell you the current state and give more details on any errors.
-
-![Update API overview](sources/api/update_api.png)
-
-First, refresh the list of available updates:
-```
-apiclient -u /actions/refresh-updates -m POST
-```
-
-Now you can see the list of available updates, along with the chosen update, according to your `version-lock` [setting](#updates-settings):
-```
-apiclient -u /updates/status
-```
-
-This will return the current update status in JSON format. The status should look something like the following (pretty-printed):
-```
-{
-  "update_state": "Available",
-  "available_updates": [
-    "0.4.0",
-    "0.3.4",
-    ...
-  ],
-  "chosen_update": {
-    "arch": "x86_64",
-    "version": "0.4.0",
-    "variant": "aws-k8s-1.15"
-  },
-  "active_partition": {
-    "image": {
-      "arch": "x86_64",
-      "version": "0.3.2",
-      "variant": "aws-k8s-1.15"
-    },
-    "next_to_boot": true
-  },
-  "staging_partition": null,
-  "most_recent_command": {
-    "cmd_type": "refresh",
-    "cmd_status": "Success",
-    ...
-  }
-}
-```
-
-You can see that we're running `v0.3.2` in the active partition, and that `v0.4.0` is available.
-If you're happy with that selection, you can request that the update be downloaded and applied to disk.  (The update will remain inactive until you make the `activate-update` call below.)
-```
-apiclient -u /actions/prepare-update -m POST
-```
-
-After you request that the update be prepared, you can check the update status again until it reflects the new version in the staging partition.
-```
-apiclient -u /updates/status
-```
-
-If the staging partition shows the new version, you can proceed to "activate" the update.
-This means that as soon as the host is rebooted it will try to run the new version.  (If the new version can't boot, we automatically flip back to the old version.)
-```
-apiclient -u /actions/activate-update -m POST
-```
-
-You can reboot the host with:
-```
-apiclient -u /actions/reboot -m POST
-```
-
 #### Updog
 
-You can also update using a CLI tool, `updog`, if you [connect through a host container](#exploration).
+You can update Bottlerocket using a CLI tool, `updog`, if you [connect through a host container](#exploration).
```

Comment:
I think this should say admin container rather than host container, and we should probably move the (now smaller) update API section above this, since we don't recommend using the admin container in production.